app/components/CoachFilterTag.tsx

```diff
@@ -40,6 +40,7 @@ export default function CoachFilterTag({
           })
         }
       />
+
```

Comment:
we can also clean up this extra line and leave it as it was before if we are not making other changes to the file, just so in the review we have fewer files to go through :)